packages/frontend/src/hooks/useColumnTotals.ts

```diff
@@ -18,20 +21,24 @@ type Args = {
     itemsMap: Record<FieldId, Field | TableCalculation>;
 };
 
-export const isSummable = (item: Field | TableCalculation) => {
-    if (isField(item) || isAdditionalMetric(item)) {
+export const isSummable = (item: Item) => {
+    if (isTableCalculation(item)) {
+        return false;
+    }
+
+    if (isField(item) || isMetric(item) || isAdditionalMetric(item)) {
```

Comment:
What's the logic for `isField` vs `isMetric` ? I think if `isField` is true then `isMetric` is always true since it's a subset of fields.

Reply:
You can replace two if statements and all the conditions inside them with:

```
if (tableCalculation) {
  return false 
}
// otherwise it's a field/metric/dimension/additionalMetric...
// do other checks
```